readme.md

```diff
@@ -32,37 +32,37 @@ brew install --cask sequel-ace
+We use Slack for communication - feel free to join our [Slack workspace](https://join.slack.com/t/sequel-ace/shared_invite/zt-g9bg1q6o-gDWyGCzauwPdg8BjmBCqKg).
 
 ### Translations
 
-If you would like to help with translations, please sign up for Crowdin and join our [Crowdin project](https://crowdin.com/project/sequel-ace) to translate Sequel Ace into supported languages. If you want to add a new language, please [creating a new issue](https://github.com/Sequel-Ace/Sequel-Ace/issues/new/choose) and we will be happy to enable that language for you to translate!
+If you would like to help with translations, please sign up for Crowdin and join our [Crowdin project](https://crowdin.com/project/sequel-ace) to translate Sequel Ace into supported languages. Also, if you want to add a new language, please [create a new issue](https://github.com/Sequel-Ace/Sequel-Ace/issues/new/choose), and we will be happy to enable that language for you to translate!
 
 ### Development
 
-If you would like to help with code and development, please see either our [projects page](https://github.com/sequel-ace/sequel-ace/projects) or [issues](https://github.com/Sequel-Ace/Sequel-Ace/issues/new/choose) with relevant labels such as [Help Wanted](https://github.com/Sequel-Ace/Sequel-Ace/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+wanted%22), [Bug](https://github.com/Sequel-Ace/Sequel-Ace/issues?q=is%3Aopen+is%3Aissue+label%3ABug), [Feature Request](https://github.com/Sequel-Ace/Sequel-Ace/labels/Feature%20Request), and [PR Welcome](https://github.com/Sequel-Ace/Sequel-Ace/labels/PR%20Welcome). These lists contain the issues where we would most like your help. There are both simple and difficult tasks there so new contributors should be able to get started.
+If you would like to help with code and development, please see either our [projects page](https://github.com/sequel-ace/sequel-ace/projects) or [issues](https://github.com/Sequel-Ace/Sequel-Ace/issues/new/choose) with relevant labels such as [Help Wanted](https://github.com/Sequel-Ace/Sequel-Ace/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+wanted%22), [Bug](https://github.com/Sequel-Ace/Sequel-Ace/issues?q=is%3Aopen+is%3Aissue+label%3ABug), [Feature Request](https://github.com/Sequel-Ace/Sequel-Ace/labels/Feature%20Request), and [PR Welcome](https://github.com/Sequel-Ace/Sequel-Ace/labels/PR%20Welcome). These lists contain the issues where we would most like your help. There are both challenging and straightforward tasks there, so new contributors should be able to get started.
 
 ## Branches
 
 - main: Main is our active development branch. All contribution PRs should be pointed at main!
 
 ## Tags & Releases
 
-- production/ tags represent submissions to the macOS App Store. These are marked as "Pre-Release" until approved by Apple and officially released by the team. Both release candidates and final releases use production/ tags. Release candidates will have RC or Release Candidate in their title.
```

Comment:
Last thing - this spot and the other similar (beta/tags) I think there should actually be a space after the /. These are indicating that beta/xxx tags are beta releases versus production/xxx tags are actual releases. I realize it's actually pretty unclear as it is, so also open to a tweak here in general but I think removing the space makes it less clear and not more clear.

After resolving those little spots, looks like some good fixes to merge in!